### PR TITLE
Disable PinClocks on stagger_u tests 

### PR DESCRIPTION
**Pull request by AlexBrownAMD**
Workaround issue with new versions of rocmsmi

### CODE REVIEW

**Review by zaliu — APPROVED** [review]

**Comment by zaliu** [comment]
I just triggered extended pipeline to see whether extended will pass.

File: Tensile/Tests/extended/stagger_u/big_skinny_B_TT.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ GlobalParameters:
   MinimumRequiredVersion: 4.4.0
   PrintLevel: 2
   CMakeBuildType: Release
-  KernelTime: True
+  KernelTime: False
   PinClocks: True
```
Comment (zaliu) [inline Tensile/Tests/extended/stagger_u/big_skinny_B_TT.yaml:6]: You meant to update line 6 to
`    PinClocks: False
`
instead I'm sure.

**Review by zaliu — APPROVED** [review]
I updated B_TT for you.  Please squash-merge.  Thanks!

**Review by cgmillette — APPROVED** [review]
LGTM

**Comment by AlexBrownAMD** [comment]
> I updated B_TT for you. Please squash-merge. Thanks!

Thanks!

**Pull request merged**